libbeat/beat/beat.go

```diff
@@ -113,7 +113,7 @@ type Beat struct {
 type BeatConfig struct {
 	Output  map[string]*common.Config
 	Logging logp.Logging
-	Shipper publisher.ShipperConfig
+	Shipper publisher.ShipperConfig `config:",inline"`
```

Comment:
Nice solution. Long term we should have all the variables in here to make it more readable.